blocks/cards/cards.css

```diff
@@ -0,0 +1,80 @@
+.block.cards {
+  display: grid;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  grid-auto-rows: 1fr;
+  gap: 1rem;
+}
+
+.block.cards .card .url {
+  position: absolute;
+  height: 100%;
+  width: 100%;
+  top: 0;
+  left: 0;
+  padding: 0;
+  opacity: 0;
+  margin: 0;
+}
+
+.block.cards .card .url > a {
+  height: 100%;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.block.cards .card {
+  position: relative;
+  background-color: white;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  text-align: center;
+  padding: 0.75rem;
+  font-size: 1rem;
+  border-radius: 16px;
+}
+
+.block.cards .card:hover {
+  box-shadow: 0 0 31px -8px rgb(10 10 10 / 30%);
+}
+
+.block.cards .card .title {
+  align-self: stretch;
+  font-weight: 700;
+
+  /* max-width: 80%; */
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.block.cards .card img {
+  width: 42px;
+  height: 42px;
+}
+
+@media (width > 767px) {
```

Comment:
we are using the defaults in boilerplate -
so for now their `767px` is our `900px`

Reply:
For 1. You have to reload the page for the layout to change

![image](https://github.com/user-attachments/assets/bfc57d3e-9d2c-4293-969c-ad6c221c6a97)
![image](https://github.com/user-attachments/assets/af391740-d039-4431-a6af-119968a1b690)

Reply:
Ah ok! sorry i was on US VPN!
that's fine then. please just update the breakpoint value then and left align the content for larger viewports